synchronicity/synchronizer.py

```diff
@@ -131,6 +135,7 @@ async def _run_generator_async(self, gen):
                 is_exc = True
 
     def _wrap_callable(self, f, return_future=None):
+        @filter_traceback
```

Comment:
isn't it enough to wrap just this function with the decorator? other methods aren't really used from the outside

Reply:
ah yup

Reply:
the ones in contextlib.py might still be needed though